tegra-bootloader-update.c

```diff
@@ -1290,6 +1290,9 @@ main (int argc, char * const argv[])
 	redundant_entry_count = nonredundant_entry_count = 0;
 	largest_length = 0;
 	memset(&mb1_other, 0, sizeof(mb1_other));
+	printf("Native TNSPEC:   %s\n", bup_tnspec(bupctx));
```

Comment:
Is this for debugging ?

Reply:
Sort of.  It's there to help with troubleshooting if something goes wrong with the BUP processing.